sdk/core/azure_core_amqp/src/connection.rs

```diff
@@ -30,32 +30,32 @@ impl AmqpConnectionOptions {
     pub fn builder() -> builders::AmqpConnectionOptionsBuilder {
         builders::AmqpConnectionOptionsBuilder::new()
     }
-    pub fn max_frame_size(&self) -> Option<u32> {
-        self.max_frame_size
+    pub fn max_frame_size(&self) -> Option<&u32> {
+        self.max_frame_size.as_ref()
```

Comment:
Why all these changes? Typically a reference (pointer) to a primitive `Copy`able is more expensive than just copying it out. It's immutable anyway.

Reply:
I removed the references for all the POD types, and left them in for the complex types.